src/status_im/keycard/common.cljs

```diff
@@ -51,7 +58,10 @@
     :no-pairing-slots))
 
 (defn tag-lost? [error]
-  (= error "Tag was lost."))
+  (or
+   (= error "Tag was lost.")
+   (= error "NFCError:100")
+   (not (nil? (re-matches #".*NFCError:100.*" error)))))
```

Comment:
`(not (nil? foo))` might be just `foo` here

Reply:
thanks, fixed!